src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java

```diff
@@ -1971,30 +2131,42 @@ public PreSubmitCallbackHandler<AsylumCase> ftpaApplicationDecisionRefusedOrNotA
             (callbackStage, callback) -> {
 
                 AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
-                boolean isAllowedOrDismissedOrRefusedOrNotAdmittedDecisionOutcome = asylumCase
-                    .read(AsylumCaseDefinition.FTPA_APPELLANT_DECISION_OUTCOME_TYPE, FtpaDecisionOutcomeType.class)
-                    .map(decision -> decision.toString().equals(FTPA_NOT_ADMITTED.toString())
-                                     || decision.toString().equals(FTPA_REFUSED.toString())
-                                     || decision.toString().equals(FTPA_REMADE32.toString()))
-                    .orElse(false);
-                if (!isAllowedOrDismissedOrRefusedOrNotAdmittedDecisionOutcome) {
-                    isAllowedOrDismissedOrRefusedOrNotAdmittedDecisionOutcome = asylumCase
-                        .read(AsylumCaseDefinition.FTPA_APPELLANT_RJ_DECISION_OUTCOME_TYPE,
-                            FtpaDecisionOutcomeType.class)
-                        .map(
-                            decision -> decision.toString().equals(FTPA_NOT_ADMITTED.toString())
-                                        || decision.toString().equals(FTPA_REFUSED.toString())
-                                        || decision.toString().equals(FTPA_REMADE32.toString()))
-                        .orElse(false);
-                }
 
                 return callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
                        && (callback.getEvent() == Event.LEADERSHIP_JUDGE_FTPA_DECISION
                            || callback.getEvent() == Event.RESIDENT_JUDGE_FTPA_DECISION)
-                       && !AsylumCaseUtils.isInternalCase(asylumCase)
```

Comment:
This was missed here. We need this internal Check.

Reply:
good spot. Added back